openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/CacheConfigAnnotation.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.openapi.converter.service;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * This class uses to store all the cache configuration details.
+ *
+ * @since 2.0.0
+ */
+public class CacheConfigAnnotation {
+    private boolean mustRevalidate = false;
+    private boolean noCache = false;
+    private boolean noStore = false;
+    private boolean noTransform = false;
+    private boolean isPrivate = false;
+    private boolean proxyRevalidate = false;
+    private int maxAge = -1;
+    private int sMaxAge = -1;
+    private List<String> noCacheFields = new ArrayList<>();
+    private List<String>  privateFields = new ArrayList<>();
+    private boolean setETag = true;
+    private boolean setLastModified = true;
+
+    public CacheConfigAnnotation() {
+
```

Comment:
Let's remove all the newlines in this class.

Reply:
Fix with this commit https://github.com/ballerina-platform/ballerina-openapi/pull/564/commits/05335d11c3cda4bfd7159122d2c3b02161a300ad